src/Symfony/Bundle/FrameworkBundle/Resources/config/services.xml

```diff
@@ -74,11 +74,5 @@
         <service id="Symfony\Component\Config\Resource\SelfCheckingResourceChecker">
             <tag name="config_cache.resource_checker" priority="-990" />
         </service>
-
-        <service id="Symfony\Component\HttpKernel\EventListener\ServiceResetListener">
```

Comment:
As discussed in #24634, we should keep that in xml, and remove it in the pass (sorry for the back and forth)

Reply:
Ok no problem :wink: Done.